main.js

```diff
@@ -11,61 +53,35 @@ function selectActivity(event) {
   var exerciseButton = document.getElementById("exercise-button");
   var exerciseImage = document.querySelector(".exercise-img");
 
-  if (event.target.id === "study-button") {
-    studySelect(studyButton, studyImage);
-    meditateDefault(meditateButton, meditateImage);
-    exerciseDefault(exerciseButton, exerciseImage);
-  } else if (event.target.id === "meditate-button") {
-    meditateSelect(meditateButton, meditateImage);
-    studyDefault(studyButton, studyImage);
-    exerciseDefault(exerciseButton, exerciseImage);
-  } else if (event.target.id === "exercise-button") {
-    exerciseSelect(exerciseButton, exerciseImage);
-    studyDefault(studyButton, studyImage);
-    meditateDefault(meditateButton, meditateImage);
+  if (event.target.classList.contains("study")) {
+    buttonSelect(studyButton, studyImage, "study");
+    buttonDefault(meditateButton, meditateImage, "meditate");
+    buttonDefault(exerciseButton, exerciseImage, "exercise");
+  } else if (event.target.classList.contains("meditate")) {
+    buttonSelect(meditateButton, meditateImage, "meditate");
+    buttonDefault(studyButton, studyImage, "study");
+    buttonDefault(exerciseButton, exerciseImage, "exercise");
+  } else if (event.target.classList.contains("exercise")) {
+    buttonSelect(exerciseButton, exerciseImage, "exercise");
+    buttonDefault(studyButton, studyImage, "study");
+    buttonDefault(meditateButton, meditateImage, "meditate");
   }
 }
 
-function studySelect(button, image) {
+function buttonSelect(button, image, category) {
   button.classList.remove("btn-default");
-  button.classList.add("study-selected");
-  image.src = "assets/study-active.svg";
+  button.classList.add(`${category}-selected`);
+  image.src = `assets/${category}-active.svg`;
 }
 
-function meditateSelect(button, image) {
-  button.classList.remove("btn-default");
-  button.classList.add("meditate-selected");
-  image.src = "assets/meditate-active.svg";
-}
-
-function exerciseSelect(button, image) {
-  button.classList.remove("btn-default");
-  button.classList.add("exercise-selected");
-  image.src = "assets/exercise-active.svg";
-}
-
-function studyDefault(button, image) {
-  button.classList.add("btn-default");
-  button.classList.remove("study-selected");
-  image.src = "assets/study.svg";
-}
-
-function meditateDefault(button, image) {
-  button.classList.add("btn-default");
-  button.classList.remove("meditate-selected");
-  image.src = "assets/meditate.svg";
-}
-
-function exerciseDefault(button, image) {
+function buttonDefault(button, image, category) {
```

Comment:
You did an excellent job refactoring six functions into one. This is a significant improvement!